openspec/changes/archive/2026-06-06-marketplace-07-module-install-state-consistency/specs/module-installation/spec.md

```
@@ -55,6 +57,8 @@ When a known module-provided command group is not registered, the system SHALL d
 - **THEN** the CLI SHALL keep reporting that the module is not installed
 - **AND** the CLI SHALL include install or init profile guidance
 
+## ADDED Requirements
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)